content/en/docs/running.md

```diff
@@ -1,156 +1,155 @@
 ---
-title: Running Falco
-weight: 2
+title: Running 
+description: Operating and Managing Falco
+weight: 4
 ---
 
-Falco is meant to be run as a service. But for experimentation and designing/testing rulesets, you will likely want to run it manually from the command-line.
 
-## Running Falco as a service
+## Run Falco as a service
 
-Once you've [installed](../installation) Falco as a package, you can start the service:
+If you installed Falco by using [the deb or the rpm](../installation) package, you can start the service:
 
 ```bash
 service falco start
```

Comment:
Can we also add a note that says that `systemctl` will also work even though the Falco service is configured with init.d

```
systemctl start falco
```

Reply:
We have to test if that works properly both on deb and rpm. Once the test has been done, I will add it.

Reply:
:point_right: https://github.com/falcosecurity/falco-website/pull/184#issuecomment-630033576